### PR TITLE
chore: remove link titles in JA

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/ja/mdn/writing_guidelines/attrib_copyright_license/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ MDN Web Docs に協力したい場合は、 Attribution-ShareAlike ライセン
 
 #### ロゴ、商標、サービスマーク、ワードマーク
 
-Mozilla Foundation の商標、ロゴ、サービスマークの権利、およびこのウェブサイトの操作画面やルックアンドフィールは、 Creative Commons ライセンスに基づいて提供されるものではなく、それらが（ロゴやグラフィックデザインといった）著作物である限り、上記の条件に基づいて許諾される著作物には含まれません。あなたが文書のテキストを利用する場合、上述の権利を行使しようとする場合、またはこのサイトにおける私たちのライセンス条件について、その他の質問がある場合は、 Mozilla Foundation の窓口 [licensing@mozilla.org](mailto:licensing@mozilla.org "mailto:licensing@mozilla.org") までご連絡ください。
+Mozilla Foundation の商標、ロゴ、サービスマークの権利、およびこのウェブサイトの操作画面やルックアンドフィールは、 Creative Commons ライセンスに基づいて提供されるものではなく、それらが（ロゴやグラフィックデザインといった）著作物である限り、上記の条件に基づいて許諾される著作物には含まれません。あなたが文書のテキストを利用する場合、上述の権利を行使しようとする場合、またはこのサイトにおける私たちのライセンス条件について、その他の質問がある場合は、 Mozilla Foundation の窓口 [licensing@mozilla.org](mailto:licensing@mozilla.org) までご連絡ください。
```
Comment (yin1999) [inline files/ja/mdn/writing_guidelines/attrib_copyright_license/index.md:48]: This title seems also exist in en-US

File: files/ja/mozilla/firefox/releases/46/index.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ translation_of: Mozilla/Firefox/Releases/46
 - ES2015 の {{jsxref("Functions/arguments/@@iterator", "arguments[@@iterator]")}} を実装しました ({{bug(1067049)}})。
 - [EcmaScript Shared Memory API](https://web.archive.org/web/20220124015148/https://tc39.es/ecmascript_sharedmem/shmem.html) を実験的に実装しました。{{jsxref("SharedArrayBuffer")}} および {{jsxref("Atomics")}} オブジェクトをご覧ください。この実験的な API を使用するには、about:config で `javascript.options.shared_memory` に `true` を設定します。
 - ECMAScript 仕様に従い、`[let](/ja/docs/Web/JavaScript/Reference/Statements/let)` および `[const](/ja/docs/Web/JavaScript/Reference/Statements/const)` によって変数を再宣言すると {{jsxref("TypeError")}} ではなく {{jsxref("SyntaxError")}} が発生するようになりました ({{bug(1198833)}})。
-- [厳格モード](/ja/docs/Web/JavaScript/Reference/Strict_mode)において、{{Glossary("primitive", "プリミティブ")}}値にプロパティを設定すると {{jsxref("TypeError")}} が発生するようになりました ({{bug(603201)}})。
+- [厳格モード](/ja/docs/Web/JavaScript/Reference/Strict_mode)において、{{Glossary("primitive",)}}値にプロパティを設定すると {{jsxref("TypeError")}} が発生するようになりました ({{bug(603201)}})。
```
Comment (yin1999) [inline files/ja/mozilla/firefox/releases/46/index.md:52]: A macro param here (and below)

File: files/ja/mdn/writing_guidelines/attrib_copyright_license/index.md (hunk 1 above)
Comment (nschonni) [reply files/ja/mdn/writing_guidelines/attrib_copyright_license/index.md:48]: Was just removed in https://github.com/mdn/content/pull/19567

**Comment by nschonni** [comment]
Marking as draft till I can rebase out a few of the macro changes that got caught in the regex

File: files/ja/mozilla/firefox/releases/46/index.md (hunk 2 above)
Comment (nschonni) [reply files/ja/mozilla/firefox/releases/46/index.md]: I've rebased out the macro ones now

**Review by mfuji09 — APPROVED** [review]
LGTM!

**Comment by mfuji09** [comment]
Thank you, @nschonni and @yin1999 !

**Pull request merged**